admin_pages/payments/templates/payment_log_details.template.php

```diff
@@ -52,10 +52,10 @@
                     </label>
                 </th>
                 <td>
-                    <?php echo $transaction
+                    <?php echo ($transaction
                         ? absint($transaction->ID())
-                        : esc_html__('Could not be determined', 'event_espresso');
-                    ?>
+                        : esc_html__('Could not be determined', 'event_espresso'));
+?>
```

Comment:
```suggestion
                    ?>
```

Reply:
I don't know why but lint throws error when PHP closing tag is there! I changed its position to the end of the current line instead of beginning of new line.

Reply:
oh weird... ok... well whatever makes the linter happy :)